src/client.ts

```diff
@@ -1165,6 +1166,18 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     return this._client.getOptimizelyConfig();
   }
 
+  /**
+   * Returns the ProjectConfig object from the underlying SDK
+   * @returns {ProjectConfig | null} optimizely project config
+   */
+  public getProjectConfig(): ProjectConfig | null {
```

Comment:
Why do we want to make it public? If we open it, it won't be easy for future change.

Reply:
That's excellent feedback. Yes. I don't see a reason to make it public.

Reply:
I removed the implementation and added it to the `Omit` on L57.